CHANGELOG.md

```diff
@@ -2,9 +2,13 @@
 
 ## Unreleased
 
+## 1.6.3
+
+- Split the `with_deferred_parent_expiration` and `with_deferred_parent_expiration`. (#578)
+
 ## 1.6.2
 
-- Support deferred expiry of associations and attributes. Add a rake task to create test database.
+- Support deferred expiry of associations and attributes. Add a rake task to create test database. (#577)
```

Comment:
Do we need to yank this release BTW? It's an unmarked breaking change. I don't really know what the process is 😅

Reply:
I also don't know the process of revoking a published release. Even we removed the changelog, the gem still exists in cloudsmith. Maybe let's just keep the record.